src/gl/interface/gl-types.ads

```diff
@@ -32,6 +32,7 @@ package GL.Types is
    type Double is new C.double;
 
    -- array types
+   type UShort_Array is array (Size range <>) of aliased UShort;
```

Comment:
You need to add `pragma Convention (C, UShort_Array);` below.

Reply:
corrected